docs/edge/Options.md.hbs

```diff
@@ -373,6 +374,10 @@ A `key: value` hash of transitions that are specific to this instance. The `key`
 
 ## Parsing
 
+> <a id="csp"></a>
+> ### csp *`Boolean`*
+> Defaults to `false`. Whether or not to add inline functions for expressions after parsing. This can effectively eliminate `eval` caused by expressions in templates. It also makes the resulting template not longer JSON compatible, so the template will have to be server from a `script` tag.
+
```

Comment:
One of these days my fingers will type what my brain tells them to without making their own minor adjustments 😆